data/getting-started/starburst.md

```diff
@@ -9,4 +9,4 @@
 * Support: 24x7 support from the Trino experts; in a fully-managed environment.
 * Get started in a matter of minutes: Starburst Galaxy improves the time to value in your data and analytics.
 
-[Sign up](https://www.starburst.io/platform/starburst-galaxy/) for the beta access and Starburst will reach out to you to grant you access to the Galaxy. Galaxy SaaS subscription is free till GA.
+**Starburst Galaxy is currently in beta**. This application is available for early access prior to official release. It won’t appear in the Enabled view, but you can access it by [signing up](https://www.starburst.io/platform/starburst-galaxy/) for beta access. Galaxy SaaS subscription is free till GA.
```

Comment:
```suggestion
**Starburst Galaxy is currently in beta**. This application is available for early access prior to official release. It won’t appear in the **Enabled** view, but you can access it by [signing up for beta access](https://www.starburst.io/platform/starburst-galaxy/). Galaxy SaaS subscription is free until GA.
```   

I made just 2 small tweaks: 
- Bolded "Enabled" so that we're aligned with how PatternFly recommends formatting referenced UI text like page titles: https://www.patternfly.org/v4/ux-writing/punctuation 
- I made the hyperlink "signing up for beta access" instead of just "signing up" for accessibility purposes (if a screenreader reads the link to users, it will make more sense when pulled out of context).

Reply:
Sounds good.